build-notifier/send_email.py

```diff
@@ -0,0 +1,43 @@
+import argparse
+import smtplib
+from google.cloud import secretmanager
+
+
+arg_parser = argparse.ArgumentParser()
+arg_parser.add_argument(
+  '--email', help='Receiver email', type=str, required=True)
+arg_parser.add_argument(
+  '--short_sha', help='Commit short sha', type=str, required=True)
+arg_parser.add_argument(
+  '--pull_request', help='Id of pull request', type=str, required=True)
+args = vars(arg_parser.parse_args())
+
+
+port = 587  # For SSL
+sender_email = 'datcom.ci@gmail.com'
```

Comment:
Should we use a google email address here (or we have datacommons@gmail.com)?

Reply:
done. using datacommonsorg@gmail.com